cloud/scope/powervs_machine.go

```diff
@@ -295,24 +506,100 @@ func (m *PowerVSMachineScope) DeleteMachine() error {
 	return nil
 }
 
-// GetBootstrapData returns the base64 encoded bootstrap data from the secret in the Machine's bootstrap.dataSecretName.
-func (m *PowerVSMachineScope) GetBootstrapData() (string, error) {
+// DeleteMachineIgnition deletes the ignition associated with machine.
+func (m *PowerVSMachineScope) DeleteMachineIgnition() error {
```

Comment:
where is this called?

Reply:
@Karthik-K-N did you drop it by mistake? I have it in my other PR, if need be I can split it in a diff PR and push.

Reply:
Updated it in the latest change to delete ignition as part of machine deletion.